default/scripting/species/SP_ACIREMA.focs.txt

```diff
@@ -5,7 +5,7 @@ Species
     Native
     CanProduceShips
 
-    tags = [ "SELF_SUSTAINING" "GREAT_INDUSTRY" "GOOD_RESEARCH" "GOOD_INFLUENCE" "GOOD_SHIELDS" "GREAT_SUPPLY" "PEDIA_SELF_SUSTAINING_SPECIES_CLASS" ]
+    tags = [ "SELF_SUSTAINING" "GREAT_INDUSTRY" "GOOD_RESEARCH" "GOOD_INFLUENCE" "ULTIMATE_SHIELDS" "GREAT_SUPPLY" "PEDIA_SELF_SUSTAINING_SPECIES_CLASS" ]
```

Comment:
Acrimea also have a bonus to planetary shields. So far the tag GOOD_SHIELDS stood for both abilities.
I suggest align the tags with the effects, so using e.g. ULTIMATE_SHIP_SHIELDS here, also also add a tag GOOD_PLANETARY_SHIELDS.

Reply:
done